README.md

```diff
@@ -157,6 +157,18 @@ Querying data stored in Parquet, CSV, JSON, Iceberg and Delta format can be done
 	LIMIT 100;
 	```
 
+Note, for Azure, you will need to first install the Azure extension:
```

Comment:
I think we should call out somewhere that writes to Azure are not yet supported, linking to this issue: https://github.com/duckdb/duckdb_azure/issues/44

Reply:
Added a note below in the readme, let me know what you think.